src/views/Derechos.vue

```diff
@@ -1,5 +1,14 @@
 <template>
   <div class="view-derechos">
-    <h1>Derechos</h1>
```

Comment:
According to design, this page have a title ("Derechos humanos fundamentales"). The title should be present, either here or in the `HumanRights` component.